server/config.yml

```diff
@@ -0,0 +1,54 @@
+---
+# dashboard server side global configuration
```

Comment:
why change config.js to config.yml

Reply:
yaml is super-set  of json,  the format is tidier. By the way, console project server side config file is also yaml

Reply:
fine